### PR TITLE
Fix screen-share tracker on Windows 10 OS

### PR DESCRIPTION
**Pull request by andrei-gavrilescu**
On Windows we need to specify `-webkit-app-region: no-drag;` for elements of a draggable window in order to interact with them, i.e. the Stop sharing button in our case.
Also from time to time some sort of provisory font is loaded before the css specified one, which caused text shift one line, giving the window more width seems to fix this problem.

### CODE REVIEW

File: screensharing/main.js [hunk 1]
```diff
@@ -72,16 +72,16 @@ class ScreenShareMainHook {
...
+            y:display.workArea.height - TRACKER_SIZE.height - 10,
             transparent: true,
             minimizable: true,
             maximizable: false,
             resizable: false,
             alwaysOnTop: true,
             fullscreen: false,
             fullscreenable: false,
-            skipTaskbar: true,
+            skipTaskbar: false,
```
Comment (saghul) [inline screensharing/main.js]: why is this change needed?

Comment (andrei-gavrilescu) [reply screensharing/main.js]: On windows if you click hide on screen share tracker, it won't appear as a minimised window in the task bar, so basically you won't be able to bring it up.
Chrome does the same thing with their screen share track window.

**Comment by andrei-gavrilescu** [comment]
> LGTM with a question. Please squash when merging.

Roger.

File: screensharing/main.js (hunk 1 above)
Comment (saghul) [reply screensharing/main.js]: Makes sense, thanks for explaining!

Comment (andrei-gavrilescu) [reply screensharing/main.js]: On mac this doesn't seem to have any effect.
